docs/supported-mutators.md

```diff
@@ -56,13 +56,15 @@ All Stryker versions support a variety of different mutators. We've aligned on a
 | MultiplicationAssignmentNegation              | `*=`                 | `/=`                 |
 | DivisionAssignmentNegation                    | `/=`                 | `*=`                 |
 | RemainderAssignmentToMultiplicationAssignment | `%=`                 | `*=`                 |
-| LeftShiftAssignmentNegation                   | `<<=`                | `>>=`                |
-| RightShiftAssignmentNegation                  | `>>=`                | `<<=`                |
+| LeftShiftAssignmentNegation                   | `<<=`                | `>>=`, `>>>=`²       |
```

Comment:
Negation can only have one outcome, not two. This should be documented as two distinct mutations

```suggestion
| LeftShiftAssignmentToRightShiftAssignment                   | `<<=`                | `>>=`, `>>>=`²       |
| LeftShiftAssignmentToUnsignedRightShiftAssignment                   | `<<=`                | `>>=`, `>>>=`²       |
```

Reply:
Took another look at the original mutator name proposal and Negation does seem to fit with the agreed upon naming so a breaking change is not worth it. I've reverted that part of the change.